core/services/standardcapabilities/delegate.go

```diff
@@ -87,6 +89,32 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 		return []job.ServiceCtx{triggerSrvc}, nil
 	}
 
+	if spec.StandardCapabilitiesSpec.Command == commandOverrideForWebAPITarget {
+		if d.gatewayConnectorWrapper == nil {
+			return nil, errors.New("gateway connector is required for web API Target capability")
+		}
+		connector := d.gatewayConnectorWrapper.GetGatewayConnector()
+		if len(spec.StandardCapabilitiesSpec.Config) == 0 {
+			return nil, errors.New("config is empty")
+		}
+		// TODO: check if TOML is the only supported format here
```

Comment:
How does this TODO get resolved?

Reply:
Will go ahead with just TOML support here. This was discussed here: https://github.com/smartcontractkit/chainlink/pull/14448/files/716f8d5334b945dde52f3d80007b8bee9ed2c116#r1762111731